packages/expo-dev-client-components/src/theme.ts

```diff
@@ -169,8 +169,8 @@ export const bg = {
 };
 
 export const bgDark = {
-  default: { backgroundColor: darkTheme.background.default },
-  secondary: { backgroundColor: darkTheme.background.secondary },
+  default: { backgroundColor: darkTheme.background.secondary },
```

Comment:
I'd like to flag this now because I think the background colors are backwards on dark mode - the background color for these buttons is "secondary" here:

<img width="370" alt="Screen Shot 2022-03-12 at 12 00 20 PM" src="https://user-images.githubusercontent.com/40680668/158033086-8d69bdad-f0c6-4d36-bf9c-c995c60400e7.png">

light mode background "default" here because secondary is the light gray you see in the background: 
<img width="363" alt="Screen Shot 2022-03-12 at 12 01 46 PM" src="https://user-images.githubusercontent.com/40680668/158033131-d388e670-f53e-4683-8a0d-19956f0b1d66.png">

Reply:
so this change lets us use the same value ("default") for these buttons to get the expected colours on each theme